### PR TITLE
Databases-week-3-muhammet-emin-alper

### PR DESCRIPTION
**Pull request by eminalpernl**

### CODE REVIEW

File: Week1/Ex-1/app.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { querier, insertData, connection } from './controllers.js';
```
Comment (ArcoMul) [inline Week1/Ex-1/app.js:1]: Could you remove the files from the homework of week 1 and 2 from this pull request?

**Review by ArcoMul — CHANGES_REQUESTED** [review]
Hi @eminalpernl, well done! One part of the mongodb exercise still has a mistake. Also please clean up the pull request by removing the files from week 1 and week 2 from this pull request. Best is to always make a new branch every week. From the main branch of the repository. You can now make a new branch from the main branch, and maybe [`cherry-pick`](https://stackoverflow.com/questions/9339429/what-does-cherry-picking-a-commit-with-git-mean) the commits you made for Week 3 into that branch.

File: Week1/Ex-1/app.js (hunk 1 above)
Comment (eminalpernl) [reply Week1/Ex-1/app.js:1]: Hi Arco sorry to respond late. I will be more carefull about deleting older files. Bu I just corrected the codes on earlier homework as you wanted. Could you take a look new version of homeworks.

File: Week3/homework/mongodb/seedDatabase.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
-const data = require("./data.json");
+import data from "./data.json" assert { type: "json" };
```
[the remark this answers is not in the document]
Comment (eminalpernl) [reply Week3/homework/mongodb/seedDatabase.js:1]: to be able to add assert statement. I added assert statement to handle an error. It was recommended in a website but I dont really remmeber what kind of an error. Maybe I could add assert statment without changing "require" statement. But I did not wanted to boteher too much:)

[the remark this answers is not in the document]
Comment (eminalpernl) [reply Week3/homework/mongodb/seedDatabase.js:1]: I think error was like this:
(node:11782) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time

File: Week3/homework/mongodb/index.js [hunk 3]
```diff
@@ -1,85 +1,135 @@
...
+        'RIVER',
+        'SNOWY_MOUNTAIN',
+        'TREE',
+        'TREES',
+      ],
+    });
 
   console.log(
-    `Created season 9 episode 13 and the document got the id ${"TODO: fill in variable here"}`
+    `Created season 9 episode 13 and the document got the id ${createOneListing.insertedID}`
```
[the remark this answers is not in the document]
Comment (eminalpernl) [reply Week3/homework/mongodb/index.js:33]: Ahh yee!!! 
I just realized that I had written "insertedID" instead of "insertedId".

**Review by ArcoMul — APPROVED** [review]
Thank you for the fix @eminalpernl, also here, files which shouldn't be in this PR can just be deleted, shouldn't be too much work. I'm approving the homework though.

**Pull request closed without merging**